### ISSUE TITLE
for AAD Account，I found AccountName is available and domainname is not available 

### ISSUE THREAD

**Issue by lijunKurosaki** [MicrosoftDocs/winrt-api#1830]
[Enter feedback here]
for AAD Account，I found AccountName is available and domainname is not available

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b1b43b0d-fdbc-11ae-d1e1-6c8cc98121de
* Version Independent ID: 98d20ec7-1f93-8e21-15d9-5b0bfa20385c
* Content: [User.GetPropertiesAsync(IReadOnlyList&lt;String&gt;) Method (Windows.System) - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/windows.system.user.getpropertiesasync?view=winrt-19041)
* Content Source: [winrt-api-build/xml/Windows.System/User.xml](https://github.com/MicrosoftDocs/winrt-api/blob/docs/winrt-api-build/xml/Windows.System/User.xml)
* Product: **uwp**
* Technology: **winrt-reference**
* GitHub Login: @hickeys
* Microsoft Alias: **hickeys**

**Comment by alvinashcraft** [comment]
Thanks for submitting your feedback and we apologize for the delayed response. I see that updates have not been made since the creation of this Issue, so I went ahead and created an internal work item to engage with our engineering folks, find the answers, and resolve this doc bug. The timeline for this may vary, so I will go ahead and close this and circle back when the changes are live.

We have moved to a new feedback system, and I want to ensure that we don't lose access to the feedback in our open issues. Though the Issue will be marked as Closed, we will monitor any replies that come afterwards so treat this like an open Issue. Many thanks.
